client/locations-list-map.js

```diff
@@ -64,6 +64,7 @@ const gDatasetMarkers = {
   },
 };
 
+// All other datasets that are not the dataset we are currently on
```

Comment:
Period at end of sentence please. :)

Though this makes me think the variable is renamed. Should it be OTHER_DATASETS?

If that can make it clear enough to not need the comment...that'd even better.

Reply:
Yeah its always filtering out the current dataset so I wonder if I can just rename it.

Reply:
please do! Better variable names >> comments.